---
authors: Freek Pols
updated: December 3, 2025
---
# Cappucino

## Introductie
Heb je al eens bedacht dat het snel verwarmen van melk voor een cappucino eigenlijk heel snel gaat zonder dat de melk verdunt wordt?
De melk wordt ook niet echt plaatselijk verhit door een warmte element.
In plaats daarvan wordt er stoom door de melk geblazen.
De stoom condenseert in de melk en geeft daarbij zijn latente warmte af.
Hierdoor warmt de melk snel op zonder dat er (significant veel) water aan toegevoegd wordt.

```{warning}
Stoom is heel heet en kan voor vervelende brandplekken zorgen.
Wees dus voorzichtig bij het uitvoeren van dit practicum.

Daarnaast, nadat de kookplaat uitgaat is er een onderdruk in de kolf.
Hang dan ook de tube uit het water wanneer je de kookplaat uitzet, anders kan de kolf imploderen.
```

In dit practicum gaan we de verdampingswarmte van water bepalen door middel van een zelfgebouwde cappucino machine.
Het principe is als volgt: We gebruiken de opstelling weergegeven in {numref}`Figuur {number} <fig_cappucino>` waarbij we water verwarmen in een kolf met behulp van een kookplaat.
Door de warmte gaat het water koken en ontstaat er stoom.
De stoom stroomt via een tube naar een afgesloten maatbeker met water.
De stoom condenseert in het maatbeker en geeft daarbij zijn latente warmte af.
Door de temperatuurstijging van het water te meten, evenals de hoeveelheid gram water dat verdampt is, kunnen we de verdampingswarmte van water bepalen.

```{figure} figures/verdampingswarmte_setup.png
:width: 70%
:label: fig_cappucino

Een schematische weergave van de cappucino opstelling.
```

## Theorie

De latente verdampinswarmte van water bedraagt 2257 kJ/kg, dit is veel meer dan de specifieke warmtecapaciteit van water (4.18 kJ/kgK).
Wanneer we de waterdamp (g) door koud water leiden, gaan we er van uit dat het waterdamp condenseert en daarbij zijn latente warmte afgeeft aan het koude water.
Door te bepalen hoeveel gram water verdampt is en hoeveel de temperatuur van het koude water stijgt, kunnen we de latente warmte van verdamping bepalen:

$$ Q_{condensatie} = m_{damp} L = m_{water} c \Delta T $$

met $m_{damp} = \Delta m_{kolf}$.

## Methoden en materialen

### Materialen
- Warmteplaat
- Kolf met stop en tube
- Maatcilinder
- Thermometer
- Weegschaal
- Water


### Procedure

- Vul de kolf met ongeveer 100 mL water: bepaal precies de massa water ($m_{w_1}$).  
- Vul de maatcilinder met ongeveer 100 mL water: bepaal precies de massa water ($m_{bad}$).
- Bepaal de temperatuur van dit waterbad ($T_{bad_1}$).
- Zet de kolf op de warmteplaat en zet de warmteplaat aan - maximale stand 3, zorg ervoor dat de tube goed in het waterbad hangt.
- Wacht tot de temperatuur van het waterbad met ongeveer 20 K is gestegen.
- Stop de meting / zet de warmteplaat uit. Noteer meteen de temperatuur($T_{bad_2}$) en haal de tube uit het water! 

```{warning}
Pas op stoom is heet!
```

- Bepaal precies de massa water in de kolf ($m_{w_2}$).
- Verwerk je resultaten hieronder om de latente warmte van verdamping van water te bepalen.

## Resultaten

In [1]:
# Met kleine maatcilinder

temp_K = 273.15             # celcius
m_kolf = 0.1146             # kg
m_w_1 = 0.2117 - m_kolf     # kg
maatbeker = 0.0301          # kg
m_bad = 0.1235 - maatbeker  # kg
T_bad_1 = 17.4 + temp_K     # K


m_w_2 = 0.2057 - m_kolf     # kg
T_bad_2 = 39.4 + temp_K     # K 39.4

c = 4.18                    # kJ/kgK
d_T = T_bad_2 - T_bad_1
d_m = m_w_1 - m_w_2

L = (m_bad * c * d_T) / d_m
print(L)

1431.5106666666654


In [2]:
# Met maatbeker

temp_K = 273.15                 # celcius
m_kolf = 0.1146                 # kg
m_w_1 = 0.2235 - m_kolf         # kg
maatbeker = 0.1892              # kg
m_bad =  0.3690 - maatbeker     # kg
T_bad_1 = 14.5 + temp_K         # K


m_w_2 = 0.2144 - m_kolf         # kg
T_bad_2 = 34.5 + temp_K         # K

c = 4.18                        # kJ/kgK
d_T = T_bad_2 - T_bad_1
d_m = m_w_1 - m_w_2

L = (m_bad * c * d_T) / d_m
print(L)

1651.7890109890113


In [3]:
# Met grote maatcilinder

temp_K = 273.15                 # celcius
m_kolf = 0.1147                 # kg
m_w_1 = 0.1992 - m_kolf         # kg
maatbeker = 0.2008              # kg
m_bad = 0.3965 - maatbeker     # kg
T_bad_1 = 16.8 + temp_K         # K


m_w_2 = 0.1899 - m_kolf         # kg
T_bad_2 = 37.9 + temp_K         # K

c = 4.18                        # kJ/kgK
d_T = T_bad_2 - T_bad_1
d_m = m_w_1 - m_w_2

L = (m_bad * c * d_T) / d_m
print(L)

1855.9514623655934


## Discussie en conclusie

De gevonden waarde bij dit experiment is 1855.9514623655934 kJ/kg. Dit wijkt sterk af van de theoretische waarde 2257 kJ/kg. Deze grote afwijking is te verklaren door de onnauwkeurige opstelling die gebruikt is voor deze metingen. De temperatuur schommelt heel erg en daardoor is het lastig een accurate start- en eindtemperatuur af te lezen. Ook de tijd tussen het stoppen van de meting en de massa aflezen beinvloed het resultaat. Ook zal het water warmte afgeven aan de omgeving waardoor het minder snel zal opwarmen. Het is ook mogelijk dat een klein deel van het gas is ontsnapt waardoor het water in de maatcilinder niet volledig opwarmt.

Ondanks de hevige afwijking laat dit experiment wel zien dat het zeer mogelijk is om de latente warmte van water te bepalen met een soortgelijke opstelling. Voor vervolgonderzoek is het aan te raden meer voorzorg te nemen om zo min mogelijk warmte aan de omgeving te verliezen. 